README.md

```diff
@@ -145,6 +145,12 @@ print('Calling Javascript...')
 eel.my_javascript_function(1, 2, 3, 4)  # This calls the Javascript function
 ```
 
+The exposed name can also be specified with in a second argument...
```

Comment:
Could you please change this line to include a reason why this might be used, e.g.

"The exposed name can also be specified by passing a second argument. You might need to use this if, you example, you minify/uglify your JavaScript source code to ensure that exposed functions can still be resolved on the Python side."

Unfortunately I can't make changes/suggestions directly on this PR, so will need you to make this change. Thanks :)

Reply:
Sure, I'll get working on that.